### PR TITLE
Config: Make TestConf DRYer, move block interval to ConsensusConfig

### PR DESCRIPTION
**Pull request by Geod24**
This contains three commits:
- First, fixing a small typo which I believe didn't have any impact but was definitely wrong;
- Second, add a `ConsensusConfig` member in `TestConf` instead of repeating its members, as was done for `NodeConfig` recently;
- Third, move `block_interval_sec` to `ConsensusConfig`, as it's already part of `ConsensusParams`;

The second commit is most of the diff, but ended up being needed to simplify the third.

### CODE REVIEW

**Review by hewison-chris — APPROVED** [review]
LGTM, although some care will be needed when resolving conflicts with first commit of PR 2270 which changes default payout_period in the tests to be 144.

**Comment by Geod24** [comment]
Rebased

**Comment by hewison-chris** [comment]
> Rebased

I checked `payout_period` and rebase looks good.

File: source/agora/test/Base.d [hunk 1]
```diff
@@ -1986,19 +1989,6 @@ public struct TestConf
 
     /// If the enrollments will be renewed or not at the end of the cycle
     bool recurring_enrollment = true;
-
-    /// The share that Validators would get out of the transction fees
-    /// Out of 100
-    public ubyte validator_tx_fee_cut = 70;
```
Comment (Geod24) [inline source/agora/test/Base.d:1992]: This was removed because it's the default value

File: source/agora/test/Base.d [hunk 2]
```diff
@@ -1944,15 +1944,6 @@ public struct TestConf
...
 
-    /// Maximum number of quorums in the autogenerated quorum sets
-    uint max_quorum_nodes = 7;
-
-    /// Overrides the default quorum threshold
-    uint quorum_threshold = 80;
-
-    /// Quorum shuffle cycle
-    uint quorum_shuffle_interval = 30;
-
```
Comment (Geod24) [inline source/agora/test/Base.d:1955]: Those are the default too

File: source/agora/test/Fee.d [hunk 3]
```diff
@@ -143,10 +141,8 @@ unittest
     import agora.common.Config;
     import core.thread;
 
-    TestConf conf = {
-        quorum_threshold : 80,
```
Comment (Geod24) [inline source/agora/test/Fee.d:148]: Default

**Comment by hewison-chris** [comment]
The test failure for unlockAge I also saw in PR https://github.com/bosagora/agora/pull/2309 and I added a commit to solve it.

**Review by linked0 — APPROVED** [review]
LGTM

**Pull request merged**